### ISSUE TITLE
Dates and tribbles

### ISSUE THREAD

**Issue by krlmlr** [cynkra/constructive#508]
``` r
library(constructive)

construct(tibble::tibble(date = Sys.Date()), opts_tbl_df("tribble"))
#> {constructive} couldn't create code that reproduces perfectly the input
#> ℹ Call `construct_issues()` to inspect the last issues
#> tibble::tribble(
#>   ~date,
#>   20069,
#> )
```

<sup>Created on 2024-12-12 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>

**Comment by moodymudskipper** [comment]
have you tried with the development version ? I can't reproduce and AFAICT this was solved by #484

**Comment by moodymudskipper** [comment]
Please reopen if issues remain
